app/services/sources/properties-managers/properties-manager.ts

```diff
@@ -60,6 +66,7 @@ export abstract class PropertiesManager implements IPropertyManager {
    * properties manager.
    */
   destroy() {
+    this.destroyed = true;
```

Comment:
this encapsulation doesn't look strong with the public instead of protected `destroyed `  property,